components/Form/ContactForm.js

```diff
@@ -225,8 +227,18 @@ function ContactForm({ onSubmit, onDelete, friendId, friend, modus }) {
             )}
           </Fragment>
         ))}
-
         <StyledButtonPosition>
+          <StyledFooter>
+            {modus === "update" ? (
+              <StyledLinkBlack
+                href={`../contacts/${friendId}-${formData.nickname}`}
+              >
+                Back
+              </StyledLinkBlack>
+            ) : (
+              <StyledLinkBlack href={`../contacts/`}>Back</StyledLinkBlack>
+            )}
+          </StyledFooter>
```

Comment:
I would like to suggest you to use abolute paths for the links. 

**Why?** 🤔Here are some advantages:
- can be clearer and more straightforward because they start from the root directory
- less dependent on the file structure
- often a good choice for larger applications where files may move around but the root directory stays the same
- can be more maintainable because if you restructure your application's directories, you won't have to change your paths

```suggestion
          <StyledFooter>
            {modus === "update" ? (
              <StyledLinkBlack
                href={`/contacts/${friendId}-${formData.nickname}`}
              >
                Back
              </StyledLinkBlack>
            ) : (
              <StyledLinkBlack href={`/contacts/`}>Back</StyledLinkBlack>
            )}
          </StyledFooter>
```

Reply:
top thx